src/darwin/Framework/CHIP/MTROTAImageTransferHandler.mm

```diff
@@ -310,7 +310,9 @@ - (instancetype)initWithMTROTAImageTransferHandler:(MTROTAImageTransferHandler *
 
     TransferSession::OutputEventType eventType = event.EventType;
 
-    ChipLogError(BDX, "OutputEvent type: %s", event.ToString(eventType));
+    if (eventType != TransferSession::OutputEventType::kNone) {
```

Comment:
I would never expect to see kNone events here, and if one does come through shouldn't we log it?  Would indicate a serious problem.

Reply:
Yeah i think i need to look at the radar to see when we got none multiple times.

Reply:
You are right. The logs in the radar are from BdxTransferDiagnosticLog.cpp not from here and that seems to not be very spammy. If needed will put a check there in a separate PR. so i will revert this.